libpimeval/src/pimCmd.cpp

```diff
@@ -500,8 +500,8 @@ pimCmdFunc1::computeRegion(unsigned index)
   unsigned numElementsInRegion = srcRegion.getNumElemInRegion();
   for (unsigned j = 0; j < numElementsInRegion; ++j) {
     if (dataType == PIM_INT8 || dataType == PIM_INT16 || dataType == PIM_INT32 || dataType == PIM_INT64 || dataType == PIM_UINT8 || dataType == PIM_UINT16 || dataType == PIM_UINT32 || dataType == PIM_UINT64) {
-      auto locSrc = locateNthElement(srcRegion, isVLayout, j, bitsPerElementSrc);
-      auto locDest = locateNthElement(destRegion, isVLayout, j, bitsPerElementDest);
+      auto locSrc = srcRegion.locateIthElemInRegion(j);
+      auto locDest = destRegion.locateIthElemInRegion(j);
```

Comment:
how are you deciding if it's vertical or horizontal? as it seems like previously we were passing it on a parameter.

Reply:
The key is numColsPerElement. For vertical layout it's 1